README.md

```diff
@@ -940,6 +941,26 @@ The answer is `3`. Surprised?
 
 * [**13.15** The `try` Statement](https://www.ecma-international.org/ecma-262/#sec-try-statement)
 
+## Asynchronous `try..catch`
+
+What message will `console.log` output?
+
+```js
+try {
+  setTimeout(function() { 
+    throw new Error('Something is going wrong..')
+  }, 1000)
+} catch (error) {
+  console.log(error)
+}
+```
+
+There will be `Uncaught Error: Something is going wrong..`
+
+### 💡 Explanation:
+
+Exceptions are only a synchronous mechanism.
```

Comment:
@ufocoder could you, please, update a PR with new examples and also add corresponding links for further reading.